service_api/grabbing_api/utils/db.py

```diff
@@ -95,9 +95,9 @@ def load(self, all_data: List[Dict]) -> None:
             try:
                 load_data(RealtyDetailsSchema(), realty_details_id, RealtyDetails)
             except KeyError as error:
-                print(error.args)
+                LOGGER.error(error.args)
             except AlreadyInDbException as error:
-                print(error)
+                LOGGER.error(error)
```

Comment:
and what about setting log level to "WARNING" here, Already in DB can be a common exception

Reply:
Why not)